.lintstagedrc

```diff
@@ -0,0 +1,9 @@
+{
+	"*.ts": [
+		"resin-lint --typescript --fix",
+		"resin-lint --typescript"
```

Comment:
I guess `resin-lint` is called twice because the first call attempts a fix, and the second call verifies that the result was successful. Sounds right?

Reply:
Yes, and the `--fix` just fixes what it can, it's possible there are things that can not be automatically fixed (eg shadowed variables)

Reply:
I've just checked and `--fix` will also report any it cannot auto-fix so I'll remove the redundant call